assets/src/app/workflows/workflows.component.ts

```diff
@@ -27,35 +26,13 @@ export class WorkflowsComponent {
     100
   ]
   video_id: string
-  after_date: undefined
-  before_date: undefined
-  page = 0
+
+  parameters: WorkflowQueryParams
+
   sub = undefined
   loading = true
   detailed = false
 
-  selectedStatus = [
-    'completed',
-    'error',
-    'processing',
-  ]
-
-  selectedWorkflows = []
```

Comment:
remove all usage of this variable, compiler Error ...

Reply:
this comment are placed on red part, it's already removed ...

Reply:
Here is the error: https://github.com/mathiaHT/ex_backend/blob/feat/workflow_statistics/assets/src/app/order/orders.component.ts#L178